app/views/home/_upcoming_deadlines.html.erb

```diff
@@ -0,0 +1,18 @@
+<div class="govuk-grid-row">
+  <h1 class="govuk-heading-l">Upcoming deadlines</h1>
+  <% current_quarter = quarter_for_date(date: Time.zone.now) %>
+  <% if current_quarter.present? %>
+    <p class="govuk-body govuk-body-m"><strong>12 January 2024</strong>: Quarterly cut off date for tenancies and sales starting between 1 October 2023 and 31 December 2023.</p>
```

Comment:
Is this one meant to be using current_quarter?

Reply:
Sure is! Thanks!